gulpfile.js

```diff
@@ -327,7 +327,8 @@ gulp.task('protractor', ['webdriver_update'], function () {
       process.exit(0);
     })
     .on('error', function(err) {
-      console.log('E2E Tests failed');
+      console.error(err);
+      console.error('E2E Tests failed');
```

Comment:
sure, in this case I'll add a colon to the `E2E Tests failed text` too to make it clear